src/reducers/systemIntakesReducer.ts

```diff
@@ -1,19 +1,45 @@
-import { STORE_SYSTEM_INTAKES } from '../constants/actions';
-import { SystemIntakesState } from '../types/systemIntake';
+import { SystemIntakesState } from 'types/systemIntake';
+import { prepareSystemIntakeForApp } from 'data/systemIntake';
+import { fetchSystemIntakes } from 'routines/routines';
+import { DateTime } from 'luxon';
 
 const initialState: SystemIntakesState = {
-  systemIntakes: []
+  systemIntakes: [],
+  loaded: false,
+  loadedTimestamp: null,
+  error: null
 };
 
 function systemIntakesReducer(
   state = initialState,
   action: any
 ): SystemIntakesState {
   switch (action.type) {
-    case STORE_SYSTEM_INTAKES:
+    case fetchSystemIntakes.TRIGGER:
       return {
         ...state,
-        systemIntakes: action.systemIntakes
+        loaded: false
+      };
+    case fetchSystemIntakes.REQUEST:
+      return state;
+    case fetchSystemIntakes.SUCCESS:
+      console.log(action);
+      return {
+        ...state,
+        systemIntakes: action.payload.map((intake: any) =>
+          prepareSystemIntakeForApp(intake)
+        )
```

Comment:
IF we were using `isLoaded` (past tense), `isLoaded` should be set here because data is only loaded when the request is successful.